### DISCUSSION TITLE
Detailed settings besides removing all window effects 移除窗口特效外，提供更细粒度的设置

### DISCUSSION THREAD

**Discussion by foresee-io** [linuxdeepin/developer-center#6374] [Features Request & Ideas | 特性请求 & 头脑风暴]
完全移除窗口动效将禁用混成器 (compositor) ，造成视觉上的卡顿。如果只是想
- 移除不必要的系统动画，如：
  - 窗口淡入淡出、缩放
  - 弹出新通知、打开“通知中心”的滚动特效
- <del>降低透明度 （已实现）</del> 

虽然通过`systemsettings`包可以设置，但对一般用户过于繁琐。GNOME、Windows、macOS都有便捷的设置入口，*通常*在 系统设置 - 辅助功能 - 视觉 内。
希望能提供便捷的 移除不必要的系统动画 开关。

**Comment by Decodetalkers** [comment]
不过以后计划不用kwin了，感觉没有精力再投入kwin的功能修改上了

**Reply by foresee-io** [reply]
确实，kwin稳定性还是差了些。之后用自研或其他框架的时候，可以考虑 把缺失的视觉无障碍功能加上去。

**Comment by BLumia** [comment]
无论是否后续换合成器，都是有必要添加相关的无障碍设置项的。cc @meiyixiang
